make.js

```diff
@@ -5,7 +5,16 @@ global.target = {};
 
 // This ensures we only execute the script targets after the entire script has
```

Comment:
this comment was meant to go right above the `setTimeout` below, otherwise it's puzzling :)